Source/WebCore/inspector/DOMPatchSupport.cpp

```diff
@@ -410,7 +408,7 @@ std::unique_ptr<DOMPatchSupport::Digest> DOMPatchSupport::createDigest(Node& nod
     SHA1 sha1;
 
     auto nodeType = node.nodeType();
-    sha1.addBytes(std::span { reinterpret_cast<const uint8_t*>(&nodeType), sizeof(nodeType) });
+    sha1.addBytes(std::bit_cast<std::array<uint8_t, sizeof(Node::NodeType)>>(nodeType));
```

Comment:
This can just call `asBytes(nodeType)` from StdLibExtras.h I think.

Reply:
I meant `asByteSpan()`, not `asBytes()`. I corrected my comment above.

Reply:
Neat! Done.